src/types/das-types.ts

```diff
@@ -324,6 +324,7 @@ export namespace DAS {
     amount?: number;
     delegated_amount?: number;
     frozen?: boolean;
+    token_extensions: any;
```

Comment:
instead of `any` could we attempt to properly type it? Here are my temp types from my codebase:

```
export interface TokenAccounts extends DAS.TokenAccounts {
  token_extensions?: TokenExtensions
}
export interface TokenExtensions {
  transfer_fee_amount: TransferFeeAmount
}

export interface TransferFeeAmount {
  withheld_amount: number
}
```

I am sure there are probably more properties but these are the ones that I have observed.

Reply:
I left it as `any` for a temp fix, but I can implement proper types here. We do have the following [`MintExtensions` type](https://github.com/helius-labs/helius-sdk/blob/a6514ebc92529df907b15ed0e48a9da6a79abbf0/src/types/das-types.ts#L358-L370) in `das-types.ts`. Would this work?

Reply:
I checked it and it does not seem to be the correct type.

Reply:
Hmm, it may be best to leave it as `any` for now and then introduce proper typing later. The goal here is to unblock users and resolve #121 as fast as possible

Reply:
sounds good

Reply:
I would suggest putting a `// TODO:` comment so we don't forget for later. I am tempted to observe the types for all of the extensions and provide them to you.